module-4/backend/week_one.md

```diff
@@ -5,16 +5,50 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+Basic Javascript syntax - makes it easier to look at. Also, the exercises for iterating were useful to complete the sorting algorithims.
+
 2. What are some tools to help debug JavaScript code?
+
+`debbuger;` will stop the code if you have the console open. `console.log` to print out to the console what you would like to see.
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+The mocha library to run tests, and the chai library to have assertions.
+
 4. What is the syntax for invoking a function?
+
+`sampleFunction()` the parentheses on the end (and any arguements needed) invokes the function. Without the parens, it won't be invoked.
+
 5. What's the difference between `==` and `===` in JavaScript?
+
+`==` checks for equality, but `===` checks for equality of both value and datatype. It is better to use `===` pretty much always, to my understanding.
+
 6. What's the difference between asynchronous and synchronous JavaScript? 
+
+Synchronous JavaScript happens in order, whereas asynchronous happens in different order. Both powerful and tricky.
```

Comment:
Be able to describe this with more nuance. Things still happen in order, but the sequence will continue before a function fully evaluates/returns a value.